system/lib/libcxxabi/src/cxa_exception_emscripten.cpp

```diff
@@ -33,27 +33,26 @@ thrown_object_from_cxa_exception(__cxa_exception* exception_header) {
 //  Get the exception object from the unwind pointer.
 //  Relies on the structure layout, where the unwind pointer is right in
 //  front of the user's exception object
-static inline __cxa_exception* cxa_exception_from_exception_unwind_exception(
+static inline __cxa_exception* cxa_exception_from_unwind_exception(
```

Comment:
This was named `cxa_exception_from_exception_unwind_exception` because it was copied from cxa_exception.cpp. I'm not sure why we need to repeat `exception`, so I shortened the name. The same with the function below.

Reply:
If this is more correct we should maybe propose this as an upstream change?    I do think there is some value in matching these names with the upstream ones... but I don't feel strongly about it.